test/e2e/config/azure-dev.yaml

```diff
@@ -214,6 +214,8 @@ variables:
   LATEST_CAPI_UPGRADE_VERSION: "v1.5.4"
   OLD_PROVIDER_UPGRADE_VERSION: "v1.12.2"
   LATEST_PROVIDER_UPGRADE_VERSION: "v1.13.0"
+  OLD_CAAPH_UPGRADE_VERSION: "v0.1.0-alpha.10"
+  LATEST_CAAPH_UPGRADE_VERSION: "v0.1.0-alpha.10"
```

Comment:
Why are these the same version? I'm assuming there isn't an older CAAPH version to use yet.

Reply:
Yep, just bumped now that we released a new version of CAAPH!